source/apps/change-deployment-models.txt

```diff
@@ -0,0 +1,208 @@
+.. _change-deployment-models:
+
+========================
+Change Deployment Models
+========================
+
+.. contents:: On this page
+   :depth: 2
+
+Overview
+--------
+
+You can change an existing App's deployment configuration to deploy it
+in a new model, region, and/or cloud provider. For a list of all
+available options, see :ref:`Deployment Models & Regions
+<deployment-model>`.
+
+To move an existing App, you begin a  **Deployment Migration** that
+automatically moves your App's data and configuration to the new
+configuration. The migration process is designed to be as seamless as
+possible but necessarily requires up to 30 minutes of downtime. You can
+monitor the migration status but no requests will be processed until the
+migration is complete. For more information, see :ref:`Deployment
+Migration Process <deployment-migration-process>`.
+
+Run a Deployment Migration
+--------------------------
+
+You can begin a deployment migration at any time. Only one migration may
+be in progress at a time. If you try to start a migration while another
+is in progress, the new migration does not run and fails with an error.
+
+Before You Begin
+~~~~~~~~~~~~~~~~
+
+.. tabs-realm-admin-interfaces::
+
+   .. tab::
+      :tabid: ui
+
+      You will need the following to modify an App's deployment model in
+      the Atlas UI:
+
+      - .. include:: /includes/prereqs/atlas-account.rst
+
+   .. tab::
+      :tabid: api
+
+      You will need the following to modify an App's deployment model
+      with the Admin API:
+
+      - .. include:: /includes/prereqs/atlas-account.rst
+
+      - .. include:: /includes/prereqs/atlas-admin-api-key.rst
+
+      - .. include:: /includes/prereqs/api-project-app-id.rst
+
+Procedure
+~~~~~~~~~
+
+.. tabs-realm-admin-interfaces::
+
+   .. tab::
+      :tabid: ui
+
+      .. procedure::
+
+         .. step:: Navigate to the Deployment screen
```

Comment:
do we usually capitalize step titles?

Reply:
i mean like title case for the whole step title, so here it'd be:

```rst
.. step:: Navigate to the Deployment Screen
```

for the next:

```rst
.. step:: Choose a Target Deployment Model and Region
```